src/index.html

```diff
@@ -7,6 +7,308 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <div class="header__logo"></div>
+
+      <nav class="navigation">
+        <ul class="navigation__items-list">
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link is-active">Apple</a>
```

Comment:
It's better to create a new block according to the BEM methodology. Then create element inside _(navigationItem__link)_ and fix the active modifier _(navigationItem__link--active)_

```suggestion
          <li class="navigationItem">
            <a href="#" class="navigationItem__link navigationItem__link--active">Apple</a>
```